format_spec/filters/webp.md

```diff
@@ -0,0 +1,48 @@
+---
+title: WEBP Filter
+---
+
+The WebP filter compresses image data using [libwebp](https://developers.google.com/speed/webp/docs/api#headers_and_libraries). The current input formats supported are `RGB`, `RGBA`, `BGR`, and `BGRA`. The colorspace format can be configured for this filter by setting the `TILEDB_WEBP_INPUT_FORMAT` filter option. Tile-based compression is determined by dimension extents. Extents equal to dimension bounds will compress the image in one pass.
+
+
+```C++
```

Comment:
Oh, I think that the C++ code goes into the examples, not in the .md spec. But @ihnorton is welcome for thoughts on this.

Reply:
Oops - I'm not sure what to replace the C++ with here, would something like this be preferred to show expected input / output value format? 

```
# Data from 2x3 pixel RGB image
input = [[255,62,83, 149,43,67, 138,43,67]
         [255,62,83, 149,43,67, 138,43,67]]
# Data is exact when we ingest and read with lossless compression
output = [[255,62,83, 149,43,67, 138,43,67]
          [255,62,83, 149,43,67, 138,43,67]]
# Data is approximate when we ingest and read with lossy compression
output = [[251,60,84, 148,51,61, 142,42,64]
          [252,63,85, 150,46,67, 139,44,68]]
```